app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,85 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+} from '@trussworks/react-uswds'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interaction
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable accounts and kafka topics creation on the GCN Kafka broker. The GCN Team is
+also happy to work with the mission teams to help construct your alerts.
+
+## Steps to Start Publishing Alerts
+
+<ProcessList>
+   <ProcessListItem>
+     <ProcessListHeading type="h3">
+       Setup Team Member Accounts
+     </ProcessListHeading>
+   - Make sure that your team member(s) who will implement GCN notices in your pipeline
+   has signed in (at least once) to https://test.gcn.nasa.gov and https://gcn.nasa.gov.
+   - Send the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+   a list of email address(es) of all the team members who should have the
+   producer access to the <code><i>mission_name</i>.\*</code> topics.
```

Comment:
Explain where these topics come from. There was probably a missing step here that the user should decide what to call their topics, and that they will be given producer access to all topics beginning with a dotted prefix.